packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -143,69 +140,83 @@ class LayoutProperties {
       children: end.children,
       constraints: BoxConstraints.lerp(begin.constraints, end.constraints, t),
       description: end.description,
-      flexFactor: begin.flexFactor + (begin.flexFactor - end.flexFactor) * t,
-      isFlex: begin.isFlex && end.isFlex,
-      size: Size.lerp(begin.size, end.size, t),
+      flexFactor: begin.flexFactor! * (t + 1) - end.flexFactor! * t,
+      isFlex: begin.isFlex! && end.isFlex!,
+      size: Size.lerp(begin.size, end.size, t)!,
       flexFit: end.flexFit,
     );
   }
 
-  LayoutProperties parent;
-  final RemoteDiagnosticsNode node;
+  LayoutProperties? parent;
+  final RemoteDiagnosticsNode? node;
   final List<LayoutProperties> children;
-  final BoxConstraints constraints;
-  final String description;
-  final num flexFactor;
-  final FlexFit flexFit;
-  final bool isFlex;
-  final Size size;
+  final BoxConstraints? constraints;
+  final String? description;
+  final num? flexFactor;
+  final FlexFit? flexFit;
+  final bool? isFlex;
+  final Size? size;
```

Comment:
can size be made non-nullable? that would eliminate several null checks and asserts elsewhere

Reply:
Done